sycl/doc/extensions/proposed/sycl_ext_oneapi_platform_device_index.asciidoc

```diff
@@ -113,30 +114,49 @@ class device {
 a@
 [source,c++]
 ----
-size_t ext_oneapi_to_index() const;
+size_t ext_oneapi_index_within_platform() const;
 ----
 |====
 
 _Returns:_ If this device is a root device as defined by the core SYCL
 specification, returns the index that it has in the `std::vector` that is
-returned when calling `device::get_devices()`.
+returned when calling `platform::get_devices()` on the platform that contains
+this device.
 
 _Throws:_ An `exception` with the `errc::invalid` error code if this device is
 not a root device.
 
 '''
 
+=== New member function in the platform class
+
+This extension adds the following new member function to the `platform` class.
+
 [frame=all,grid=none,separator="@"]
 |====
 a@
 [source,c++]
 ----
-static device ext_oneapi_from_index(size_t index);
+class platform {
+  // ...
+  device ext_oneapi_index_to_device(size_t index) const;
```

Comment:
Nit; I am not fully sold on this name. Maybe `ext_oneapi_device_at(size_t index)` or `ext_oneapi_device_at_index(size_t index)`?

Not a strong preference though.